src/Sylius/Bundle/CoreBundle/Form/Type/CartItemType.php

```diff
@@ -34,7 +34,7 @@ public function buildForm(FormBuilderInterface $builder, array $options)
     {
         parent::buildForm($builder, $options);
 
-        if (isset($options['product']) && $options['product']->hasVariants()) {
+        if (isset($options['product']) && $options['product']->hasVariants() && !$options['product']->isSimple()) {
```

Comment:
Not related to this PR, but `$options['product']` will always exist (see `configureOptions`, so we can get rid of `isset($options['product'])`.